examples/golang/put-bucket-notification-creation/put-bucket-notification-creation.go

```diff
@@ -57,13 +57,24 @@ func main() {
 
 	svc := s3.New(sess)
 
+	filterRules := make([]*s3.FilterRule, 0, 1) // slice with type, length and capacity
```

Comment:
can you create a list of filter rules without dynamic allocation?
see here: https://go.dev/blog/slices-intro
as for how to init a slice in glonag

Reply:
I made a suffix rule example. Can I also include one with a prefix?